x/dex/keeper/asset_list.go

```diff
@@ -0,0 +1,41 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/sei-protocol/sei-chain/x/dex/types"
+)
+
+func (k Keeper) SetAssetMetadata(ctx sdk.Context, assetMetadata types.AssetMetadata) {
+	store := ctx.KVStore(k.storeKey)
+	// Even if asset exists already, overwrite the store with new metadata
```

Comment:
Maybe add a comment saying that we only allow one asset per base for now

Reply:
Added on latest commit